src/main/java/org/candlepin/controller/ContentAccessManager.java

```diff
@@ -465,45 +477,8 @@ else if (type.isType(ConsumerTypeEnum.HYPERVISOR)) {
         return entitlementVersion != null && entitlementVersion.startsWith("3.");
     }
 
-    /**
-     * Fetches the path to use for content used to generate the ENTITLEMENT_DATA certificate
-     * extension
-     *
-     * @param owner
-     *  the owner of the entitlement for which the content path is being generated
-     *
-     * @param environment
-     *  the environment to which the consumer receiving the entitlement belongs
-     *
-     * @return
-     *  the container content path for the given owner and environment
-     */
-    private String getContainerContentPath(Owner owner, Environment environment) {
-        // Fix for BZ 1866525:
-        // - In hosted, SCA entitlement content path needs to use a dummy value to prevent existing
-        //   clients from breaking, while still being clear the path is present for SCA
-        // - In satellite (standalone), the path should simply be the content prefix
-
-        if (this.standalone) {
-            String envId = environment == null ? null : environment.getId();
-            List<Environment> envs = environment == null ? List.of() : List.of(environment);
-            return ScaContentPrefix.from(owner, this.standalone, envs).get(envId);
-        }
-        return "/sca/" + Util.encodeUrl(owner.getKey());
-    }
-
     private String createDN(Consumer consumer, Owner owner) {
-        StringBuilder sb = new StringBuilder("CN=")
-            .append(consumer.getUuid())
-            .append(", O=")
-            .append(Rdn.escapeValue(owner.getKey()));
-
-        if (consumer.getEnvironmentId() != null) {
-            Environment environment = this.environmentCurator.getConsumerEnvironment(consumer);
-            sb.append(", OU=").append(Rdn.escapeValue(environment.getName()));
-        }
-
-        return sb.toString();
+        return "CN=" + consumer.getUuid() + ", O=" + Rdn.escapeValue(owner.getKey());
```

Comment:
Using StringBuilder instead of raw Strings would be more appropriate here.

Reply:
With recent Java (8 or higher) versions JVM internally make use of StringBuilder when we try to concatenation simple string just like above. So we can skip this comment.